packages/foam-vscode/src/features/link-completion.ts

```diff
@@ -197,7 +197,20 @@ export class CompletionProvider
       item.range = replacementRange;
       item.command = COMPLETION_CURSOR_MOVE;
       item.commitCharacters = linkCommitCharacters;
-      return item;
+      const aliasItems = resource.aliases.map(a => {
+        const item = new ResourceCompletionItem(
+          a.title,
+          vscode.CompletionItemKind.Reference,
+          resource.uri
+        );
+        item.insertText = this.ws.getIdentifier(resource.uri) + '|' + a.title;
+        item.detail = `Alias of ${resource.uri.getName()}`;
```

Comment:
Like we do for resources, I would show `vscode.workspace.asRelativePath(toVsCodeUri(resource.uri));`